detector/aws_alb_invaid_security_group.go

```diff
@@ -10,22 +10,20 @@ import (
 
 type AwsALBInvalidSecurityGroupDetector struct {
 	*Detector
-	IssueType      string
-	TargetType     string
-	Target         string
-	DeepCheck      bool
 	securityGroups map[string]bool
 }
 
 func (d *Detector) CreateAwsALBInvalidSecurityGroupDetector() *AwsALBInvalidSecurityGroupDetector {
-	return &AwsALBInvalidSecurityGroupDetector{
+	nd := &AwsALBInvalidSecurityGroupDetector{
 		Detector:       d,
-		IssueType:      issue.ERROR,
-		TargetType:     "resource",
-		Target:         "aws_alb",
-		DeepCheck:      true,
 		securityGroups: map[string]bool{},
 	}
+	nd.Name = "aws_alb_invalid_security_group"
+	nd.IssueType = issue.ERROR
+	nd.TargetType = "resource"
+	nd.Target = "aws_alb"
```

Comment:
__[gometalinter]__
* [goconst] 2 other occurrence(s) of "aws_alb" found in: aws_alb_invalid_subnet.go:24:14 aws_alb_duplicate_name.go:23:14

<!-- Comment by [SideCI](https://sideci.com) -->